pushapkscript/docker.d/worker.yml

```diff
@@ -55,12 +55,19 @@ products:
               certificate_alias: dep
 
       'COT_PRODUCT == "mobile" && ENV == "prod"':
-        - product_names: [ "fenix"]
+        - product_names: [ "fenix" ]
           digest_algorithm: 'SHA-256'
           skip_check_multiple_locales: true
           skip_check_same_locales: true
           skip_checks_fennec: true
           apps:
+            fennec-nightly:
+              package_names: [ "org.mozilla.fennec_aurora" ]
+              certificate_alias: 'fennec-nightly'
+              google:
+                default_track: 'alpha'
+                service_account: { "$eval": "GOOGLE_PLAY_SERVICE_ACCOUNT_FIREFOX_AURORA" }
```

Comment:
We should add these two variables within https://github.com/mozilla-releng/scriptworker-scripts/pull/136/files#diff-2236ae2520a422cd49a56083c288edfeR88 to make sure we fail fast if they are not present there.

Reply:
Great catch! I had forgotten about this. Fixed!